precompiles/wasmd/wasmd.go

```diff
@@ -247,7 +247,7 @@ func (p Precompile) query(ctx sdk.Context, method *abi.Method, args []interface{
 }
 
 func (p Precompile) getRemainingGas(ctx sdk.Context) uint64 {
-	gasMultipler := p.evmKeeper.GetPriorityNormalizer(ctx)
+	gasMultiplier := p.evmKeeper.GetPriorityNormalizer(ctx)
 	seiGasRemaining := ctx.GasMeter().Limit() - ctx.GasMeter().GasConsumedToLimit()
-	return new(big.Int).Mul(new(big.Int).SetUint64(seiGasRemaining), gasMultipler.RoundInt().BigInt()).Uint64()
+	return sdk.NewDecFromInt(sdk.NewIntFromUint64(seiGasRemaining)).Quo(gasMultiplier).RoundInt().Uint64()
```

Comment:
previous: `seiGasRemaining * gasMultipler`
after: `seiGasRemaining / gasMultiplier`

I think it should be divided by `gasMultiplier` here because gas limit was multiplied by `gasMultiplier` already in `RunAndCalculateGas()`.